src/test/java/com/flightstats/hub/metrics/StatsDFilterTest.java

```diff
@@ -21,6 +23,7 @@
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
 class StatsDFilterTest {
```

Comment:
Since this setting is required only for some tests, it would be nice to add at method level
Mockito.lenient().when(mock.foo(1)).thenReturn(1);

Reply:
Cool, I didn't know about that (and didn't look too closely because I really just wanted to stabilize the build). I'll swap it out.